distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -1151,6 +1151,38 @@ Be sure to note the port number that replaced 8181 then enter that number in the
 Only root can access ports<1024 on Unix systems. For suggested ways to run {branding} with ports < 1024 see How do I use port 80 as a non-root user?.
 ====
 
+=== Configuring {branding} using Configuration (.config) files
+
+The {branding} can also be configured with configuration (.config) files.
+
+[NOTE]
+====
```

Comment:
@oconnormi: This is probably where we should say that only security related things can be configured using the new configuration file format. Otherwise people may try to use the new format for bundles that haven't been changed yet.